contrib/parseq-retry/src/main/java/com/linkedin/parseq/retry/termination/NeverTerminate.java

```diff
@@ -0,0 +1,23 @@
+package com.linkedin.parseq.retry.termination;
+
+
+/**
+ * A termination policy that never signals for termination.
+ *
+ * @author Oleg Anashkin (oleg.anashkin@gmail.com)
+ */
+public class NeverTerminate implements TerminationPolicy {
```

Comment:
Someone might needed it for simple side projects. I'll add a warning to javadoc discouraging usage of this policy.